data/core.yaml

```diff
@@ -1812,6 +1812,11 @@ en:
           one: "{highway} is disconnected from other roads and paths"
           other: "{count} routable features are connected only to each other"
         reference: "All roads, paths, and ferry routes should connect to form a single routing network. This feature (or small collection of features) is either disconnected from the routing network or only connected to it via a feature that few applications consider routable."
+    empty_start_date:
+      title: Empty start date
```

Comment:
Let’s call this a “missing” start date, for consistency with other rules like “Missing Roles” and “Missing Tags”. In principle, we could group these issues with the existing “Missing Tags” issues, though I see the value in splitting it out separately, since it’ll be a lot noisier and individual warnings will be more difficult to address than the usual missing tag warnings.